cnd/cnd.ui/src/org/netbeans/modules/cnd/resources/templates/qt/resource.template

```diff
@@ -1,4 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
```

Comment:
Are these user templates? If so, this looks as if the license header is taken over into the generated file. I would not add a header then.